src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4012,8 +4012,8 @@ namespace EnergyPlus::ZoneEquipmentManager {
                                     Infiltration(MassConservation(ZoneNum).InfiltrationPtr).MassFlowRate =
                                         max(0.0, Infiltration(MassConservation(ZoneNum).InfiltrationPtr).MassFlowRate);
                                 } else {
-                                    MassConservation(ZoneNum).InfiltrationMassFlowRate =
-                                        Infiltration(MassConservation(ZoneNum).InfiltrationPtr).MassFlowRate;
+                                    Infiltration(MassConservation(ZoneNum).InfiltrationPtr).MassFlowRate = 0.0;
+                                    MassConservation(ZoneNum).InfiltrationMassFlowRate = 0.0;
```

Comment:
When infiltration treatment is AdjustInfiltrationFlow, the adjusted infiltration must be set to zero if the infiltration term calculated less than the convergence tolerance limit.

Reply:
I disagree with this. The mass balance should allow `MassConservation(ZoneNum).InfiltrationMassFlowRate` to be negative (or create a new variable for `MassConservation(ZoneNum).ExfiltrationMassFlowRate`) to allow a pressurized zone to be balanced.

Reply:
The code change still allows negative infiltration flow rates (exfiltration flow rates). Rather this change simply sets the infiltration flow rate to zero if flow rates is smaller than the flow tolerance limits. Note that the code checks for "std::abs(ZoneInfiltrationMassFlowRate) > ConvergenceTolerance".

Reply:
OK, well in an effort to try to get #8460 merged in for IO freeze, I'm going to table this discussion for right now, but I would like to see this discussed further on the next review call to make sure we all feel comfortable with the changes before 9.5 goes out.